Sources/Datadog/DatadogInternal/DD.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// Core utilities for monitoring performance and execution of the SDK.
+/// These are meant to be shared by all instances of the SDK and `DatadogCore`.
+internal var dd = DD()
+
+/// Bundles core utilities for monitoring performance and execution of the SDK.
+internal struct DD {
+    /// The logger providing methods to print debug information and execution errors from Datadog SDK to user console.
+    ///
+    /// It is meant for debugging purposes when using the SDK, hence **it should log information useful and actionable
+    /// to the SDK user**. Think of possible logs that we may want to receive from our users when asking them to enable
+    /// SDK verbosity and send us their console log.
+    var logger: CoreLoggerType
+
+    // TODO: RUMM-2239 Move `Telemetry` in here
```

Comment:
WDYT of moving `Telemetry` in here @maxep ? By doing this, we'd need to also enhance `RUMTelemetry` to support lazy binding and buffering messages before "bind" is established, e.g.:
```swift
var telemetry: Telemetry = LazyTelemetry()
```
and later:
```swift
(dd.telemetry as? LazyTelemetry).bind(rumTelemetry)
```

We could discuss it offline or through RFC if you think it's worth considering multiple angles 🙌.

Reply:
I'm up for having `telemetry` here as well 👍 

The binding would be a great enhancement, so we can record pre-initialisation telemetry entries. But I guess we could already add the `telemetry` in here, either as optional or with a no-op default implementation. And tackle the lazy telemetry later. WDYT?

(The binding would also be useful when we support multiple core instances)

Reply:
Great 💪. I'll move `Telemetry` in next PR(s), without yet focusing on lazy binding.